java/src/jmri/jmrit/display/layoutEditor/PositionablePoint.java

```diff
@@ -194,7 +196,8 @@ protected LayoutEditor getLayoutEditor() {
     }
 
     @CheckReturnValue
-    public @Nonnull String getEastBoundSignal() {
+    public @Nonnull
```

Comment:
In these instances where you have ```public @Nonnull```, you want to reverse them to ```@Nonnull public``` so that when you format the code, the ```public``` is on the same line as the method declaration.